wrappers/src/dictionary_cs.cpp

```diff
@@ -226,4 +227,11 @@ extern "C" {
         });
     }
 
+    REALM_EXPORT Results* realm_dictionary_get_filtered_results(const object_store::Dictionary& dictionary, uint16_t* query_buf, size_t query_len, realm_value_t* arguments, size_t args_count, NativeException::Marshallable& ex)
+    {
+        return handle_errors(ex, [&]() {
+            realm::Results values = dictionary.get_values();
```

Comment:
it's safe to refer to `get_values()` directly right? No need to create a new Ream Results from this?

Reply:
@nirinchev what do you think about this?

Reply:
Yeah, don't see a reason to allocate it on the heap.